src/main/java/kg/giftlist/giftlist/apis/WishApi.java

```diff
@@ -1,4 +1,57 @@
 package kg.giftlist.giftlist.apis;
 
+import io.swagger.v3.oas.annotations.tags.Tag;
+import kg.giftlist.giftlist.dto.SimpleResponse;
+import kg.giftlist.giftlist.dto.wish.WishRequest;
+import kg.giftlist.giftlist.dto.wish.WishResponse;
+import kg.giftlist.giftlist.models.User;
+import kg.giftlist.giftlist.services.impl.WishServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/wish")
+@CrossOrigin
+@Tag(name = "WISH API", description = "Any user can create, update or delete wishes!")
 public class WishApi {
```

Comment:
Please refactor the class, there are unnecessary lines also add endpoint descriptions